### ISSUE TITLE
Get attr value with named attr

### ISSUE THREAD

**Issue by gwendall** [lapwinglabs/x-ray#35]
I'm unsuccessfully trying to extract meta tags from a page. For instance the following returns no description. Any idea of what is wrong with my selector?

```
var url = "http://techcrunch.com";
var selection = {
  $root: 'head',
  title: 'title',
  description: 'meta[name="description"][content]'
};

xray(url)
.select(selection)
.run(function(err, res) {
  console.log(err, res);
});

**Comment by matthewmueller** [comment]
Sorry about that! That is available in the new selector engine (https://github.com/lapwinglabs/x-ray-select), I'm working on a  new version of x-ray so I haven't had a chance to make it work with this version. 

I'm accepting PRs to make this version work with the new engine though.

**Comment by jmartsch** [comment]
I was looking for excactly the same thing but with [itemprop="streetAddress"].
+1
Can we use x-ray with the new selector engine right now?

**Comment by vishalkrv** [comment]
Facing the same issue. I m in need of meta description tag and the itemprop as well. Is there any hack to get it working with the current selector?

**Comment by jmartsch** [comment]
As an alternative you can try https://github.com/rc0x03/node-osmosis which support this kind of selectors:
```
'city': 'span[itemprop="addressLocality"]'
```

**Comment by matthewmueller** [comment]
This is now fixed in 1.0.0 :-)
